10.9-libcxx/texinfo.patch

```diff
@@ -0,0 +1,529 @@
+@@ -0,0 +1,29 @@
++.TH INSTALL-INFO 1
++.\" NAME should be all caps, SECTION should be 1-8, maybe w/ subsection
++.\" other parms are allowed: see man(7), man(1)
++.SH NAME
++install-info \- wrapper around GNU install-info
++.SH SYNOPSIS
++.B install-info
++.I "[options]"
++.br
++.SH DESCRIPTION
++This manual page documents briefly the
++.BR install-info
++command.
++This manual page was written for the Debian GNU/Linux distribution
++because the original script was designed for Debian packaging system.
++.PP
++.B install-info
++is a wrapper around GNU install-info. If it is called from a normal
++shell or script it acts like GNU install-info by forwarding all options
++to ginstall-info.
++.PP
++If it is called from a maintainer script it gives a warning to rebuild
++packages and does nothing, since info file installation is now done
++via triggers.
++.SH OPTIONS
++Same as with GNU install-info, see ginstall-info(1).
++.SH AUTHOR
++This manual page was written by Norbert Preining <preining@logic.at>,
++for the Debian GNU/Linux system (but may be used by others).
+diff -ruN texinfo-6.3.orig/fink/install-info.sh texinfo-6.3/fink/install-info.sh
+--- texinfo-6.3.orig/fink/install-info.sh	1969-12-31 17:00:00.000000000 -0700
++++ texinfo-6.3/fink/install-info.sh	2016-10-11 08:53:59.000000000 -0600
+@@ -0,0 +1,36 @@
++#!/bin/sh
++#
++# Wrapper to the GNU's install-info, to be compatible with the one that used to
++# be packaged by dpkg on Debian.
++#
++# written by Norbert Preining, this is not copyrightable ;-)
++#
++set -e
++
++if [ -z "$DPKG_RUNNING_VERSION" ] ; then
++  # it seems we are running from outside a maintainer script, so give a
++  # warning and call ginstall-info without anything else
++  echo "This is not dpkg install-info anymore, but GNU install-info" >&2
++  echo "See the man page for ginstall-info for command line arguments" >&2
++  @FINKPREFIX@/bin/ginstall-info "$@"
++else 
++  # we are running from a maintainer script, simply ignore the call
++  # since we have trigger support and people should rebuild their
++  # package with new debhelper which does not add calls to install-info
++  # Do not complain if called with "--remove" or "--remove-exactly",
++  # as these are used in old packages' prerm scripts (see #546165)
++  while [ -n "$1" ]; do
++    case "$1" in
++      --remove|--remove-exactly)
++        exit 0
++        ;;
++      *)
++        shift
++        ;;
++    esac
++  done
++  echo "Ignoring install-info called from maintainer script" >&2
++  echo "The package $DPKG_MAINTSCRIPT_PACKAGE should be rebuilt with new debhelper to get trigger support" >&2
++fi
++
++
+diff -ruN texinfo-6.3.orig/fink/install-info.triggers texinfo-6.3/fink/install-info.triggers
+--- texinfo-6.3.orig/fink/install-info.triggers	1969-12-31 17:00:00.000000000 -0700
++++ texinfo-6.3/fink/install-info.triggers	2016-10-11 08:53:59.000000000 -0600
+@@ -0,0 +1,2 @@
++interest-noawait @FINKPREFIX@/share/info
++interest-noawait @FINKPREFIX@/info
```

Comment:
Do we need to be scanning @FINKPREFIX@/info ? This path is explicitly forbidden by the validator.

Reply:
This was just to be safe, it doesn't hurt anything if it's never used.

Reply:
Fair enough.